apps/meteor/ee/app/message-read-receipt/server/hooks/afterReadMessages.ts

```diff
@@ -0,0 +1,24 @@
+import type { IUser, IRoom, IMessage } from '@rocket.chat/core-typings';
+
+import { Reads } from '../../../../server/sdk';
```

Comment:
Can't you import it from @rocket.chat/core-services?

Reply:
Done! Thanks for suggesting this, though (I had forgotten to add this service to the `core-services` package :grimacing: )